src/createCatalog.ts

```diff
@@ -29,26 +31,26 @@ function getComponents(components: any): any {
 }
 
 function getChapters(first: any, second: any): any {
-  if (validateCatalogDataFiles(first) && validateCatalogDataFiles(second)) {
-    return first.chapters.concat(second.chapters);
+  if (first) {
+    if (validateCatalogDataFiles(first) && validateCatalogDataFiles(second)) {
+      return first.chapters.concat(second.chapters);
+    }
+  } else {
+    if (validateCatalogDataFiles(second)) {
+      return second.chapters;
+    }
   }
 }
 
 function getStandards(first: any, second: any): any {
-  if (validateCatalogDataFiles(first) && validateCatalogDataFiles(second)) {
-    return first.standard.concat(second.standard);
-  }
-}
-
-function getTitle(data: any): any {
```

Comment:
Why did getTitle & getLang get nixed here?

Reply:
I was lazily building it from the data files, instead I now pass it when calling `createCatalog` from the librarian file.